src/cr/cube/cube_slice.py

```diff
@@ -252,15 +266,65 @@ def is_double_mr(self):
         return self.dim_types == ['multiple_response'] * 2
 
     def scale_means(self, hs_dims=None, prune=False):
+        """Calculate scale means of the CubeSlice."""
```

Comment:
It's a fine point, but I always use the word "Return" as the first word of the docstring for a method that returns a value. So something like "Return scale-means for this cube slice." This is the key aspect of the contract of such a method (the return value and type) and what the caller cares about most. Saying "Calculate" could just indicate a side-effect and leaves the reader to dig into the code (which often they won't have when just looking at the docs).

As an aside, in the case of a property, I leave the 'Return' word out, like "int count of foos in bar" instead of 'Return int count of foos in bar', since the semantics of a property are a value rather than a procedure.

You'll also need to say what scale-means are and what form you deliver them in. If that's not compact enough for the summary line, then in a body paragraph of the docstring. Even then, it's worth every effort not to repeat the method name in the summary because it provides no additional information. It could be something like """Return list containing scaled-mean for each slice column.""" or something and already starts to give more insight into the method contract.